gravitee-apim-console-webui/src/management/api/policy-studio/debug/components/policy-studio-debug-inspector/policy-studio-debug-inspector-text/policy-studio-debug-inspector-text.component.ts

```diff
@@ -14,20 +14,38 @@
  * limitations under the License.
  */
 
-import { Component, Input } from '@angular/core';
+import { Component, Input, OnChanges, SimpleChanges } from '@angular/core';
+
+import { isAdded, isDeleted, isUpdated } from '../policy-studio-debug-inspector.component';
 
 @Component({
   selector: 'policy-studio-debug-inspector-text',
   template: require('./policy-studio-debug-inspector-text.component.html'),
   styles: [require('./policy-studio-debug-inspector-text.component.scss')],
 })
-export class PolicyStudioDebugInspectorTextComponent {
+export class PolicyStudioDebugInspectorTextComponent implements OnChanges {
   @Input()
-  private name: string;
+  name: string;
 
   @Input()
-  private input: string;
+  input: string;
 
   @Input()
-  private output: string;
+  output: string;
+
+  classes: {
+    added: boolean;
+    deleted: boolean;
+    updated: boolean;
+  };
+
+  ngOnChanges(changes: SimpleChanges): void {
+    if (changes['input'] || changes['output']) {
+      this.classes = {
```

Comment:
this.diffClass =  isAdded(this.input, this.output) ? 'added': isUpdated(...) ? 

or others 🤷‍♂️